packages/js/src/axiomClient.ts

```diff
@@ -0,0 +1,260 @@
+import { Axiom } from "./client";
+import { ClientOptions } from "./httpClient";
+
+
+interface LogEvent {
+  level: string;
+  message: string;
+  fields: any;
+  _time: string; 
+}
+
+enum LogLevel {
+  debug = 0,
+  info = 1,
+  warn = 2,
+  error = 3,
+  off = 100,
+}
+
+export type LoggerConfig = {
+  args?: { [key: string]: any };
+  logLevel?: LogLevel;
+  autoFlush?: boolean;
+  source?: string;
+  req?: any;
+  token: string | undefined;
+  dataset: string | undefined;
+};
+
+
+const Version = 'AXIOM_VERSION';
```

Comment:
What is this?

Reply:
When running the Rollup build, replace AXIOM_VERSION with the version. I noticed that we were already doing that because we don't have access to the 'fs' module.